packages/address/src/tests/AddressFormatter.test.ts

```diff
@@ -143,7 +143,7 @@ describe('getCountries()', () => {
   it('should not call the API again for the countries if the locale is the same.', async () => {
     mockAPICall('countries', countriesEn, 'YY');
 
-    const addressFormatter = new AddressFormatter('YY');
+    const addressFormatter = new AddressFormatter('yy' as SupportedLocale);
```

Comment:
Can you just add a comment on the line above saying `// Bypass the cache by using a non existant locale`?

Reply:
Same for the other instances as well.